### PR TITLE
Update SETUP.md

### PR DESCRIPTION
**Pull request by srkunze**
couldn't pull upstream:

Permission denied (publickey).
fatal: Could not read from remote repository.
Please make sure you have the correct access rights and the repository exists.

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
@LukasReschke @jancborchardt can one of you add @srkunze to the organization please? :)

**Comment by AndyScherzinger** [comment]
@srkunze can you pull upstream in case of using https: instead of git: ?

**Comment by srkunze** [comment]
@AndyScherzinger Indeed. Don't know what's different to the git: protocol but using https: I could pull.

My .git/config
```
[core]
	repositoryformatversion = 0
	filemode = true
	bare = false
	logallrefupdates = true
[remote "origin"]
	url = https://github.com/srkunze/android.git
	fetch = +refs/heads/*:refs/remotes/origin/*
[branch "master"]
	remote = origin
	merge = refs/heads/master
[remote "upstream"]
	url = https://github.com/nextcloud/android.git
	fetch = +refs/heads/*:refs/remotes/upstream/*
[branch "folder-based-sort-order"]
	remote = origin
	merge = refs/heads/folder-based-sort-order
```

**Comment by AndyScherzinger** [comment]
Strange, but whatever solves the issue ;) I can't use git either but that is due to my employer's network...

**Comment by strugee** [comment]
`git@` is SSH shorthand, which GitHub requires you to have set up in advance. (I.e. you need an SSH key [added to your account](https://github.com/settings/keys) to even read repositories). On the other hand `https://` is, well, HTTPS, which GitHub lets people use to read repos without any kind of login.

`git://`, on the gripping hand, is a custom Git protocol. IIRC it's basically SSH but without all the crypto. GitHub allows anonymous Git protocol access too but I wouldn't recommend it because the protocol itself(?) doesn't support writes and also can be trivially tampered with because it's basically plaintext :)

**Comment by mario** [comment]
Um, pretty sure you can write via git protocol. Anyway, irrelevant ... The point is we should rather tell people to setup their keys properly for git@ than move to https. Or mention both and let people choose :P

**Comment by AndyScherzinger** [comment]
I'd say mention both since for example people on corporate networks (like me throughout the day) can't use git but have to use https.
So having both described lowers the barrier... :)

**Comment by srkunze** [comment]
@strugee As soon as mentioned it, I now remember git: being based on ssh:. So, it definitely needs some kind of setup on the users machine.

@AndyScherzinger I like the simplicity of only proposing only one way. https: just works for both push and pull. How would you phrase this passage with both protocols?

**Comment by AndyScherzinger** [comment]
I am totally fine with just describing the https way to keep the barrier at a minimum and leave out git. While pros will know how to set it up using git and likely don't need this Infos at all.

**Comment by jancborchardt** [comment]
Yep - simple instructions which just work are better than educating people. I think this change is good :+1:

**Comment by AndyScherzinger** [comment]
Sweet! @srkunze would you mind changing the other git url to https also? That way everything should then be described for https:)

**Comment by srkunze** [comment]
Done. :)

**Comment by mario** [comment]
👍

**Comment by AndyScherzinger** [comment]
:+1:

**Comment by AndyScherzinger** [comment]
Merged 🎉

File: SETUP.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Next steps will assume you have a Github account and that you will get the code
 
 * In a web browser, go to https://github.com/nextcloud/android, and click the 'Fork' button near the top right corner.
 * Open a terminal and go on with the next steps in it.
-* Clone your forked repository: ```git clone --recursive git@github.com:YOURGITHUBNAME/android.git```.
+* Clone your forked repository: ```git clone --recursive https://github.com:YOURGITHUBNAME/android.git```.
```
Comment (comradekingu) [inline SETUP.md]: There shouldn't be a ":" after github.com when using HTTPS.

Should be `git clone --recursive https://github.com/YOURGITHUBNAME/android.git`

**Comment by comradekingu** [comment]
See #1499

**Pull request merged**